ci-templates/gitlab/image-builder.yml

```diff
@@ -0,0 +1,108 @@
+# SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+# SPDX-License-Identifier: Apache-2.0
+
+variables:
+  FRONTEND:
+    value: "1"
+    description: "Build the frontend image?"
+  BACKEND:
+    value: "1"
+    description: "Build the backend image?"
+  DOCS:
+    value: "1"
+    description: "Build the docs image?"
+  GUACAMOLE:
+    value: "0"
+    description: "Build the guacamole image?"
+  REVISION:
+    value: "main"
+    description: "Revision of Github repository"
+  DOCKER_BUILD_ARGS: "--no-cache"
+  DOCKER_BUILDKIT: "1"
+
+default:
+  image: $DOCKER_REGISTRY/base
```

Comment:
Should we use a more commonly used base image (e.g. `debian`)?

Reply:
It's as trivial, as the `docker`, `node`, `python`, `git` and `npm` have to be installed. And we don't want to install it in each pipeline run. I did a bit of research, but haven't found any matching image. Do you know an image where these tools are already preinstalled?

Reply:
Not really. Let's keep it like this for now, then. If somebody from the community runs into this, he/she may come up with a proper solution.